src/anemoi/training/train/forecaster.py

```diff
@@ -263,16 +319,59 @@ def calculate_val_metrics(
         y: torch.Tensor,
         rollout_step: int,
         enable_plot: bool = False,
-    ) -> tuple[dict, list]:
+    ) -> tuple[dict, list[torch.Tensor]]:
+        """Calculate metrics on the validation output.
+
+        Args:
```

Comment:
Please make sure to set your docstring style to numpy

Reply:
Copied from ensemble codebase, will address